client/append_test.go

```diff
@@ -52,29 +50,27 @@ func TestAppendToStreamSingleEventNoStream(t *testing.T) {
 	container := GetEmptyDatabase()
 	defer container.Close()
 
-	client := CreateTestClient(container, t)
-	defer client.Close()
-	testEvent := messages.ProposedEvent{
-		EventID:      uuid.FromStringOrNil("38fffbc2-339e-11ea-8c7b-784f43837872"),
-		EventType:    "TestEvent",
-		ContentType:  "application/octet-stream",
-		UserMetadata: []byte{0xd, 0xe, 0xa, 0xd},
-		Data:         []byte{0xb, 0xe, 0xe, 0xf},
-	}
-	proposedEvents := []messages.ProposedEvent{
-		testEvent,
-	}
+	db := CreateTestClient(container, t)
+	defer db.Close()
 
-	streamID, _ := uuid.NewV4()
+	testEvent := createTestEvent()
+	testEvent.EventID = uuid.FromStringOrNil("38fffbc2-339e-11ea-8c7b-784f43837872")
+
+	streamID := uuid.Must(uuid.NewV4())
 	context, cancel := context.WithTimeout(context.Background(), time.Duration(5)*time.Second)
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(5)*time.Second)
```

collide with the package name